assets/js/modules/analytics/components/setup/SetupFormGA4Transitional.stories.js

```diff
@@ -206,6 +206,9 @@ export default {
 				registry
 					.dispatch( MODULES_ANALYTICS )
 					.selectAccount( accountID );
+				registry
+					.dispatch( MODULES_ANALYTICS_4 )
+					.selectAccount( accountID );
```

Comment:
Same here, I do not think this is necessary. Besides, this component isn't used and will not likely be recreated for `analytics-4`.